ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/ParticipantMode.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils.app
+
+sealed abstract class ParticipantMode
+
+object ParticipantMode {
+
+  /** Run the full participant */
+  case object Full extends ParticipantMode
```

Comment:
What about calling this `Combined`? I.e., that would imply we are running different types of services. On a side note: regardless, just an indexer doesn't make a participant yet.